tools/src/main/scala/scala/scalanative/linker/Classpath.scala

```diff
@@ -19,22 +19,20 @@ sealed trait Path {
   def globals: Set[Global]
 }
 
-object Path {
+object Classpath {
```

Comment:
@jonas Using Java 7 APIs is fine or sbt io utilities as convenient. I was just trying to point out the state of affairs with the sbt versions etc. The sbt folks are part of the same community so reach out if needed. You guys are doing great stuff so carry on.